src/SamplesApp/SamplesApp.UITests/QueryExtensions.cs

```diff
@@ -49,6 +49,38 @@ public static IAppRect GetRect(this IApp app, string elementName)
 		{
 			return app.WaitForElement(elementName).Single().Rect;
 		}
+		public static IAppRect GetRect(this IApp app, QueryEx query)
+		{
+			return app.WaitForElement(query).Single().Rect;
+		}
+		public static IAppRect GetRect(this IApp app, Func<IAppQuery, IAppQuery> query)
+		{
+			return app.WaitForElement(query).Single().Rect;
+		}
+
+		public static void FastTap(this IApp app, string elementName)
+		{
+			var tapPosition = app.GetRect(elementName);
+			app.TapCoordinates(tapPosition.CenterX, tapPosition.CenterY);
+		}
+
+		public static void FastTap(this IApp app, QueryEx query)
+		{
+			var tapPosition = app.GetRect(query);
+			app.TapCoordinates(tapPosition.CenterX, tapPosition.CenterY);
+		}
+
+		public static void FastTap(this IApp app, Func<IAppQuery, IAppQuery> query)
+		{
+			var tapPosition = app.GetRect(query);
+			app.TapCoordinates(tapPosition.CenterX, tapPosition.CenterY);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'app' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4926142)